components/DateThumbnail.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { View, Text } from 'react-native';
+
+
+interface Props {
+    coverSource: Date;
+    newsTitle: string;
+    newsDuration: string;
+}
+
+const DateThumbnail: React.FC<Props> = ({ coverSource, newsTitle, newsDuration }) => {
+    return (
+        <View className={'flex-start m-2'}>
+            <View className={'flex justify-center bg-gray-200 rounded-lg w-32 h-32'}>
+              <Text className={'text-primary text-center font-extrabold mt-2'}>
+                <Text className={'text-5xl'}>{coverSource.getDate()+"\n"}</Text>
+                <Text className={'text-xl'}>{coverSource.toLocaleDateString('default', {month: 'short'})}</Text>
+              </Text>
+            </View>
+            <Text className={'w-32 mt-2 text-16'}>{newsTitle}</Text>
+            <Text className={'mt-2 font-bold text-14'}>{newsDuration}</Text>
```

Comment:
Ik denk dat newsTitle niet echt relevant is hier. Het gaat toch in principe over de podcast van een bepaalde datum? Minder text is beter hier denk ik